src/Game/Managers/BoatMovingManager.cs

```diff
@@ -64,7 +64,7 @@ private static int GetVelocity(byte speed)
 
                 case 0x04: return FAST_INTERVAL;
 
-                case > 0x04: return speed;
+                case > 0x04: int Custom_Interval = speed * 10; return Custom_Interval;
```

Comment:
speed * 10 implicity creates an integer, you don't need to created a new variable.

Reply:
I originally just had it at speed * 10, but it threw errors in the build (macos-latest) check, creating a new variable gave it the green light